Makefile.am

```diff
@@ -542,6 +542,18 @@ amqp_la_LIBADD = \
 	libformat_json.la
 endif
 
+if BUILD_PLUGIN_AMQP1
+pkglib_LTLIBRARIES += amqp1.la
+amqp1_la_SOURCES = src/amqp1.c
```

Comment:
You also need to add `src/utils_deq.h` or else it won't be included in the tarball which is used for building RPMs.

```
amqp1_la_SOURCES = src/amqp1.c \
        src/utils_deq.h
```